### PR TITLE
Mmoser/html css fix ns

### PR DESCRIPTION
**Pull request by markmoser**
Some misc changes/refactors

1. header containers with fixed header nav get the ```.with-fixed-header``` class to help adjust heights.
1. cleaned up html for legal page with proper containers and grid
1. added an offset to anchor destinations ```:target``` so they do not scroll up under the header and provide a little more head space when no fixed header

### CODE REVIEW

File: src/css/elements/_typography.scss [hunk 1]
```diff
@@ -16,11 +16,30 @@ h6 {
...
 }
 
-h1 { font-size: 3rem; }
-h2 { font-size: 2.375rem; }
-h3 { font-size: 1.875rem; }
-h4 { font-size: 1.25rem; }
-h5 { font-size: 1rem; }
+h1,
+.h1-size {
+  font-size: 3rem;
```
Comment (scytherswings) [inline src/css/elements/_typography.scss]: whoah I've got some learning to do. this frontend stuff gets crazy real fast. Here I was thinking we just had `%` `px` and `em` and maybe another, but now you're throwing American rock bands at me? Idk if I am cut out for this stuff. 

:laughing:

**Comment by scytherswings** [comment]
Forgot that Joel wasn't on here!

File: src/css/elements/_typography.scss (hunk 1 above)
Comment (markmoser) [reply src/css/elements/_typography.scss]: 🎸 👨‍🎤 👩‍🎤 🎸

**Comment by stretch1414** [comment]
I approve

**Pull request merged**